app/upgrades/mainnet/v5/upgrade.go

```diff
@@ -0,0 +1,25 @@
+package v5
+
+import (
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	liquiditykeeper "github.com/crescent-network/crescent/v4/x/liquidity/keeper"
+	liquiditytypes "github.com/crescent-network/crescent/v4/x/liquidity/types"
+)
+
+const UpgradeName = "v5"
+
+func UpgradeHandler(
+	mm *module.Manager, configurator module.Configurator,
+	liquidityKeeper liquiditykeeper.Keeper) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		liquidityKeeper.SetMaxNumMarketMakingOrdersPerPair(ctx, liquiditytypes.DefaultMaxNumMarketMakingOrdersPerPair)
+
+		return mm.RunMigrations(ctx, configurator, fromVM)
```

Comment:
I think `RunMigrations` should be executed first to set paramstore, and after that, we can set `SetMaxNumMarketMakingOrdersPerPair`

Could you add test code for the upgrade handler?

Reply:
After applying the change above I think the upgrade handler would become simple as:
```go
func UpgradeHandler(
	mm *module.Manager, configurator module.Configurator) upgradetypes.UpgradeHandler {
	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
		return mm.RunMigrations(ctx, configurator, fromVM)
	}
}
```